environment/container/fileinfo_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+ *  Copyright IBM Corporation 2021
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package container
+
+import (
+	"io/fs"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+)
+
+func TestFileInfoName(t *testing.T) {
+	t.Run("Testing Name method", func(t *testing.T) {
+		// Create a FileInfo instance with a specific name
+		expectedName := "testfile.txt"
+		fileInfo := &FileInfo{
+			stat: types.ContainerPathStat{
+				Name: expectedName,
+			},
+		}
```

Comment:
lets try getting `types.ContainerPathStat` from any simple operation rather than constructing it ourselves and then pass it. We are not actually testing much here by constructing the struct ourselves.

Reply:
lets try the same (https://github.com/konveyor/move2kube/pull/1138#discussion_r1468882073) for all of your other test functions in this PR. Thanks.

Reply:
Got it working on the requested changes